spec/migrations/20160317041206_add_maintenance_to_host_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_migration
```

Comment:
this spec is not needed, we don't usually test changes in structure, only when you would do a data change